scripts/tables.py

```diff
@@ -0,0 +1,182 @@
+import builtins
+import datetime
```

Comment:
Importing submodules instead of these parent modules breaks the match statement below as dot notation allows the use of the types/classes we actually want to use `str` and `datetime`